### PR TITLE
[IMP] hr_timesheet,_*: improve generic ux for timesheet

### PR DESCRIPTION
**Pull request by kcv-odoo**
So in this PR improve the form view, filter and report and
some feature usability.

Purpose of this PR to improve the generic UX of the timesheet app.

- Add the measures in the timesheet/attendance report.
- Currently, the timesheet manager cannot modify the invoiced timesheet. Will be able to now.

task-2782768

### CODE REVIEW

File: addons/project_timesheet_holidays/models/hr_holidays.py [hunk 1]
```diff
@@ -32,6 +34,12 @@ def _compute_timesheet_generate(self):
         for leave_type in self:
             leave_type.timesheet_generate = leave_type.timesheet_task_id and leave_type.timesheet_project_id
 
+    @api.depends('timesheet_task_id', 'timesheet_task_id.project_id')
+    def _compute_timesheet_project_id(self):
+        for leave_type in self:
+            if not leave_type.timesheet_project_id:
+                leave_type.timesheet_project_id = leave_type.timesheet_task_id.project_id
```
Comment (xavierbol) [inline addons/project_timesheet_holidays/models/hr_holidays.py:41]: What do we do in the case in which we have a project and a task set on a leave type and the task is linked to another project than the one set on the leave type? 🤔

**Review by xavierbol — COMMENTED** [review]
Thanks for your work, I have made a comment. 🙂

File: addons/project_timesheet_holidays/models/hr_holidays.py (hunk 1 above)
Comment (kcv-odoo) [reply addons/project_timesheet_holidays/models/hr_holidays.py]: Hello @xavierbol 
Skip this point from spec by @aju-odoo.
Thanks!

**Review by IT-Ideas — APPROVED** [review]
Thanks 🙂 
@robodoo r+

**Comment by IT-Ideas** [comment]
@robodoo r+

**Pull request closed without merging**